nextjs/src/app/(app)/data-sources/ExternalDataSourceList.tsx

```diff
@@ -117,6 +118,9 @@ function PageHeader() {
         <div className="rounded-3xl bg-meepGray-700 px-10 py-6 overflow-hidden flex flex-row items-center justify-center">
           <AirtableLogo className="w-full" />
         </div>
+        <div className="rounded-3xl bg-meepGray-700 px-10 py-6 overflow-hidden flex flex-row items-center justify-center">
+          <MailChimpLogo className="w-full" />
+        </div>
```

Comment:
Let's give this a maximum height so that its dimensions are somewhat similar to the other two logos.

Reply:
have done. I was thinking it's a placeholder anyway as I just grabbed it off the internet. @Arbyhisenaj might have one he prefers to use